roles/auth_configure/tasks/user_auth_userdefined.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Task to setup the Auth
+
+- name: Setup Auth
+  shell: "/usr/lpp/mmfs/bin/mmuserauth service create --data-access-method file --type userdefined"
+  ignore_errors: yes
+  when: is_admin_node | default(false) == true and service_result.rc != 0
```

Comment:
I don't think we need to have `is_admin_node`  flag in the when condition. you can run this from one of the protocol node .

Reply:
Resolved